plenum/server/replica.py

```diff
@@ -1477,6 +1491,13 @@ def stashCheckpoint(self, ck: Checkpoint, sender: str):
             self.stashedRecvdCheckpoints[seqNoStart, seqNoEnd] = {}
         self.stashedRecvdCheckpoints[seqNoStart, seqNoEnd][sender] = ck
 
+    def stashed_checkpoints_with_quorum(self):
+        quorums_num = 0
```

Comment:
Can't it be
return len([ for key, senders in self.stashedRecvdCheckpoints.items() if self.quorums.checkpoint.is_reached(len(senders.keys()))])
?